pages/linux/homectl.md

```diff
@@ -0,0 +1,36 @@
+# homectl
+
+> Create, remove, change or inspect home directories using systemd-homed service.
+> More information: <https://wiki.archlinux.org/index.php/Systemd-homed>.
```

Comment:
I wonder if we should use the man [Arch man page](https://man.archlinux.org/man/homectl.1) for the link, instead the current link, because though it's great to link to the documentation for the whole portable user accounts business, the page barely documents `homectl`; in fact, it demonstrates the simplest invocation of the command and nothing more.

The man page, on the other hand, fully documents all of the options and their valid values, and even provides an [examples section](https://man.archlinux.org/man/homectl.1#EXAMPLES) at the bottom (which you could look at for more context).
```suggestion
> More information: <https://man.archlinux.org/man/homectl.1>.
```

Reply:
Sometimes I wonder if we should line the same output than `man homectl`, but the Arch Wiki explains another stuff around this command.